### DISCUSSION TITLE
Connect to GoPro

### DISCUSSION THREAD

**Discussion by minhson0506** [gopro/OpenGoPro#82] [Q&A] [answered]
Hello.
I have problem with connect to GoPro
After pairing between phone and GoPro, how can I send AP command for opening WIFI?
Thanks

**Comment by tcamise-gpsw** [comment, marked as answer]
Hello. Please see the tutorial here: https://gopro.github.io/OpenGoPro/tutorials/python/connect-wifi

**Reply by MaHDiaLaGaB** [reply]
i download and install the API and when i write this command 
>>>pip show open-gopro
i can find that everything is find but i went through the first tutorial connect by BLE i couldn't find my camera at all even if i run the demos like
>>>gopro-photo
its keep scanning with no results i am using Gopro hero9 and i have mac air m1
